docs/conf.py

```diff
@@ -19,18 +19,19 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
+# list all readthedocs env. vars
+for item, value in os.environ.items():
+    print("[READTHEDOCS] {} = {}".format(item, value))
 
 # -- Project information -----------------------------------------------------
 
 project = u'FireSim'
 copyright = u'2018, Sagar Karandikar, Howard Mao, Donggyu Kim, David Biancolin, Alon Amid, and Berkeley Architecture Research'
 author = u'Sagar Karandikar, Howard Mao, Donggyu Kim, David Biancolin, Alon Amid, and Berkeley Architecture Research'
 
-# The short X.Y version
-version = u''
-# The full version, including alpha/beta/rc tags
-release = u''
-
+# The short X.Y version (given by rtd)
```

Comment:
Instead of X.Y, can you provide a couple examples here under for tags, and the behavior under stable + latest